Makefile

```diff
@@ -22,8 +22,19 @@ init:
 
 	until docker exec $(POSTGRES_CONTAINER_NAME) pg_isready ; do sleep 5 ; done
 
+	sleep 5
+
+	until docker exec $(POSTGRES_CONTAINER_NAME) pg_isready ; do sleep 5 ; done
+
```

Comment:
Why do you add this? Isn't it pointless bcs previous loop in line 23 is already doing the same and is resulting in True. Is it because when postgres initialise and is ready then postgis initialises and causes database to don't accept connections ? So after getting postgres up we want to wait for postgis to also activate ?

Reply:
Without this `make rebuild` causes errors, maybe you see some better way to fix it? I will try to find easier fix to this problem

Reply:
I think this might solve the issue https://github.com/HIM-Services/heaven-in-mouth/pull/1/files#r1701755947
You can try to implement this and this should solve the issue

Reply:
I think `make` shouldn't be used inside Makefile commands

Reply:
Thanks for tips, im new to makefiles and docker itselft so I really appreciate this feedback. I will fix it in few minutes.